packages/service-clients/azure-client/api-report/azure-client.api.md

```diff
@@ -67,7 +67,7 @@ export interface AzureContainerVersion {
 
 // @internal @deprecated
 export class AzureFunctionTokenProvider implements ITokenProvider {
-    constructor(azFunctionUrl: string, user?: Pick<AzureMember<any>, "name" | "id" | "additionalDetails"> | undefined);
+    constructor(azFunctionUrl: string, user?: Pick<AzureMember<any>, "name" | "additionalDetails" | "id"> | undefined);
```

Comment:
This could cause your PR checks to fail, I'm assuming that the build machines end up with "id" first when they generate this, and then it'll fail the build because the file changed.  Pretty annoying.